parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleReadSupport.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.tools.read;
+
+
+import java.io.StringWriter;
+import java.io.PrintWriter;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.apache.parquet.schema.MessageTypeParser.parseMessageType;
+import static org.apache.parquet.hadoop.metadata.CompressionCodecName.UNCOMPRESSED;
+import org.apache.hadoop.fs.Path;
+import org.apache.parquet.example.data.simple.SimpleGroupFactory;
+import org.apache.parquet.hadoop.example.GroupWriteSupport;
+import org.apache.parquet.schema.MessageType;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.parquet.hadoop.ParquetWriter;
+import org.apache.parquet.hadoop.ParquetReader;
+import org.apache.parquet.hadoop.example.ExampleParquetWriter;
+import org.apache.parquet.example.data.Group;
+import org.apache.parquet.hadoop.example.GroupReadSupport;
+
+public class TestSimpleReadSupport {
```

Comment:
This unit test fails in the Travis build: testNullValues(org.apache.parquet.tools.read.TestSimpleReadSupport): optional fixed_len_byte_array(3) bar is not a group

Reply:
seems a change in master broke, this. I built against master and get a similar error when running the `cat` tool: ```☁  parquet-tools [master] java -jar target/parquet-tools-1.10.1-SNAPSHOT.jar  cat -debug -j /tmp/foobar/part-00000-1ca7e0f8-30ab-4ce9-860d-a791e2df081e-c000.snappy.parquet  
java.lang.ClassCastException: required int32 element is not a group
```

I will investigate a fix